google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseId.java

```diff
@@ -94,6 +94,11 @@ public static DatabaseId of(String project, String instance, String database) {
     return new DatabaseId(new InstanceId(project, instance), database);
   }
 
+  /** Creates a {@code DatabaseId} given instance and database IDs. */
+  public static DatabaseId of(String instance, String database) {
+    return new DatabaseId(new InstanceId("default", instance), database);
```

Comment:
I think we should document to users that this method assumes instance Id to be "default".

Reply:
maybe not instance Id, but rather project Id?